react/rules.tsx

```diff
@@ -11,23 +11,36 @@ export interface PhoneRuleDescriptor {
    * Mask to the phone. E.g.: "(99) 9 9999-9999"
    */
   mask?: string
+  /**
+   * Regex used for the phone validation
+   */
+  pattern: string
 }
 
+/**
+ * Default rules for the phone field. Don't use this
```

Comment:
What do you mean by "don't use this"?

Reply:
I forgot to add the rest of the comment 😅 We won't be maintaining/updating this list anymore, so I wanted to warn the devs that see this file of it.

Reply:
Oh. I guess that answers my other questions as well :^)